src/nav/infrastructure.yml

```diff
@@ -165,6 +165,20 @@ pages:
             path: /docs/infrastructure/prometheus-integrations/install-configure-openmetrics/add-rename-or-copy-prometheus-attributes
           - title: Mutual TLS authentication
             path: /docs/infrastructure/prometheus-integrations/install-configure-openmetrics/add-mutual-tls-prometheus-endpoints
+      - title: Install and configure New Relic Prometheus Configurator
+        pages:
+          - title: Install New Relic's Prometheus Agent Configurator
+            path: /docs/infrastructure/prometheus-integrations/install-configure-prometheus-configurator/install-prometheus-configurator
```

Comment:
We use three terms to refer to the same product: `prometheus-agent` (chart name), "Prometheus Configurator", and "Prometheus Agent Configurator". I think we should use only one.

Reply:
This is still being discussed. @x8a will take care of that. I made the first rename but she is going to make it consistent.